bindings/generator/generate.py

```diff
@@ -513,12 +563,20 @@ def generate_client(api_specs: ApiSpecs) -> list[str]:
     return []
 
 
+def generate_client_common(api_specs: ApiSpecs) -> list[str]:
```

Comment:
instead of having yet another .ts file, we can just rename `definition.d.ts` into `definition.ts`

This way we can put enum along with the typing stuff:
```ts
export const enum OS {
    Linux = 'OSLinux',
    MacOs = 'OSMacOs',
    Windows = 'OSWindows',
    Android = 'OSAndroid',
}

export type Result<T, E = Error> =
  | { ok: true; value: T }
  | { ok: false; error: E }

[...]
```

Reply:
done